docs/pages/admin-guides/management/operations/ca-rotation.mdx

```diff
@@ -28,23 +28,32 @@ following order:
 1. `init`: A new certificate authority is issued, but not used.
 1. `update_clients`: The Teleport Auth Service uses the new CA to sign
    certificates but continues to trust certificates signed by the original CA.
-1. `update_servers`: Teleport cluster components (Agents, Auth Service, and
-   Proxy Service instances) reload and start serving TLS and SSH certificates
-   signed by the new certificate authority, but still accept certificates issued
-   by the original certificate authority. This only applies to the Teleport
-   [`host` CA](#host).
-   1. `standby`: No rotation in progress. All operations have completed.
+1. `update_servers`: Any server components in the cluster that accept incoming
+   connections from clients should reload their identity and start serving
```

Comment:
What does "should" mean here? Would the sentence make sense if we removed it, or are there some situation in which cluster components that accept incoming connections do not reload their identity etc.?

Reply:
Well, in some cases (OpenSSH servers, self-hosted databases), it's on the admin to issue a new cert and reload the identity, that's why I used "should"